definition/data/sheets/AuthorisationCaseField.json

```diff
@@ -204,6 +204,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "evidenceHandled", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paperResponseType", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "paperResponseType", "UserRole": "citizen", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseFieldID": "state", "UserRole": "caseworker-cmc", "CRUD": "CU"},
```

Comment:
why we are adding state In case

Reply:
@dharmendrak there is an issue with CCD that for the About To Submit call back if we are going to change the state, we have to pass it as the case detail. So to overcome we have to have this field. I explained to @ckalista and he had taken it to CCD. They confirmed that its existing ticket.